features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/data/AuthManager.jsx

```diff
@@ -177,8 +177,13 @@ class AuthManager {
         if (user) {
             const userScopes = user.scopes;
             const validScope = APIClient.getScopeForResource(resourcePath, resourceMethod);
-            return validScope.then((scope) => {
-                return userScopes.includes(scope);
+            let validity = []
+            return validScope.then((scopes) => {
+                scopes.map(scope => {
```

Comment:
Declaring `validity` array outside the `then` callback function scope is not valid, 

You can refer to [this implementation](https://github.com/wso2/carbon-apimgt/pull/10404/files#diff-108a02ceb37d6f854480c966d632c0f0f671dea4df9b417fca76271d2676d2ebR89-R95)